README.md

```diff
@@ -45,3 +45,61 @@ make l1-remove
 - Update the `L1_CHAIN_ID` with the value of the `network_id` in the `network_params` section
 - Update the `L1_RPC_URL` to `http://localhost:<EL_PORT_START + 2>`, where `EL_PORT_START` is the value of `public_port_start` in the `el` section
 - Update the `L1_BEACON_URL` to `http://localhost:<CL_PORT_START + 1>`, where `CL_PORT_START` is the value of `public_port_start` in the `cl` section
+
+## Launch OP Stack L2
+
+### Set the environment variables
+
+```bash
+cp .env.example .env
+cp .env.explorer.example .env.explorer
+cp .env.bridge.example .env.bridge
+```
+
+For the local L1, the L1 chain ID and pre-funded account private key can be retrieved from the L1 server (see steps above).
+- `L1_CHAIN_ID`
+- `L1_FUNDED_PRIVATE_KEY`
```

Comment:
we shouldn't just copy from the above sections and leave with two copies. it will cause confusion.

they should be cleaned up